009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,443 @@
+    \item  \textbf{Extraversion}: The extent to which an individual is willing and eager to socialise and interact with other individuals. An extroverted individual is more likely to occupy positions of leadership and influence. The extraversion score affects the responsiveness. A high extraversion score increases the likelihood of positive social interaction such as accepting requests or treaties. 
+    \item  \textbf{Agreeableness}: An individual with high agreeableness will exhibit greater degrees of kindness, trust and altruism. Whereas an individual with low agreeableness will demonstrate a lack of sympathy and may be spiteful or manipulative of others. Therefore, greediness and kindness are initialised by agreeableness and scaled accordingly. Agreeableness also has influence over responsivenes in the same way that openness and extraversion do.
+    \item  \textbf{Neuroticism}: The neuroticism trait dictates the emotional stability of an individual. It is associated with greater proneness to anxiety, irritability and mood swings. In the extreme case, individuals can experience regular fluctuations in their character. Therefore, an agent that is high in Neuroticism will commit actions that may seem unreasonable or illogical. The way this is implemented is that neuroticism will create random day-to-day volatality in the greediness and kindness levels of an agent. Since greediness and kindness directly control the amount of food taken, an agent with higher Neuroticism may either intend to take an inadequate or excess amount of food.
+\end{itemize}
+
+Any time a new agent is deployed it is initialised with a value for these traits ranging from 20-80. We did this in order to exclude extreme personality traits on either end. The personality traits assigned to an agent are innate and unchanged for the duration of the simulation. This is done in order to emulate how traits remains fairly constant over a person's life.
+
+\subsection{Behavioural Traits}
+\label{subsec: Behaviour}
+In contrast to personality traits, behavioural characteristics of an agent change over the course of the simulation. The behaviours are initialized based on the agent's personality traits and can range from 0-100 over the lifetime of the agent. These behaviours are then updated daily by environmental factors such as food availability, floor changes and interactions with other agents. 
+
+The behaviour characteristics are described below:
+\begin{itemize}
+    \item  \textbf{Greediness}: This is the degree to which an agent behaves selfishly, specifically with regards to the amount of food it takes. The greediness value is initialised by the agent's agreeableness score meaning that high agreeableness will result in low initial greediness and vice versa. The greediness value is subsequently adjusted throughout the simulation run by the following factors: 
```

Comment:
since you mention the relationship between greediness and the other variables here, I don't think you have to mention it above since it is confusing